app/controllers/api/v1/offers_controller.rb

```diff
@@ -139,6 +139,13 @@ def mark_inactive
         render json: @offer, serializer: offer_serializer
       end
 
+      def summary
+        priority_and_non_priority_active_offers_count = Offer.priority_and_non_priority_offers_count_for(self_reviewer: false).merge(
```

Comment:
if the function returns the priority count AND the non priority count, isn't that the same as all count ?? 😄 
Therefore shouldn't the function be named `offers_count_for` ?

Reply:
actually that comment applies more to the implementation of the method below (poorly placed)